dependencies.gradle

```diff
@@ -11,48 +11,46 @@ ext {
     // publish correctly a new version of each of our libraries.
     // e.g. screenManager library, you could have as a current version
     // 1.0.0-beta1, if you want to publish a new version you will have to increase in one the
-    //screenManagerCoreVersionCode and to bump the screenManagerCoreVersionName.
+    // screenManagerCoreVersionCode and to bump the screenManagerCoreVersionName.
 
     // SurfaceDuo SDK libraries version code.
     // If you want to publish a new version, bump in one (1) the specific line(s)
     screenManagerCoreVersionCode = 4
     screenManagerDMVersionCode = 4
     screenManagerWMVersionCode = 4
     screenManagerUtilsVersionCode = 4
-    layoutsVersionCode = 7
-    fragmentsHandlerVersionCode = 4
-    bottomNavigationVersionCode = 3
-    tabsVersionCode = 3
-    recyclerviewVersionCode = 5
+    layoutsVersionCode = 8
+    fragmentsHandlerVersionCode = 5
+    bottomNavigationVersionCode = 4
+    tabsVersionCode = 4
+    recyclerviewVersionCode = 6
     inkSDKVersionCode = 6
-    wmUtilsVersionCode = 3
-    navigationVersionCode = 1
+    wmUtilsVersionCode = 4
+    navigationVersionCode = 2
 
     // SurfaceDuo SDK libraries version name:
     // If you want to publish a new version, bump the specific line(s)
     screenManagerCoreVersionName = '1.0.0-beta4'
     screenManagerDMVersionName = '1.0.0-beta4'
     screenManagerWMVersionName = '1.0.0-beta4'
     screenManagerUtilsVersionName = '1.0.0-beta4'
-    layoutsVersionName = '1.0.0-beta7'
-    fragmentsHandlerVersionName = '1.0.0-beta4'
-    bottomNavigationVersionName = '1.0.0-beta3'
-    tabsVersionName = '1.0.0-beta3'
-    recyclerviewVersionName = '1.0.0-beta5'
-    inkSDKVersionName = "1.0.0-alpha4"
-    wmUtilsVersionName = "1.0.0-beta3"
-    navigationVersionName = "1.0.0-alpha1"
+    layoutsVersionName = '1.0.0-beta8'
+    fragmentsHandlerVersionName = '1.0.0-beta5'
+    bottomNavigationVersionName = '1.0.0-beta4'
+    tabsVersionName = '1.0.0-beta4'
+    recyclerviewVersionName = '1.0.0-beta6'
+    inkSDKVersionName = "1.0.0-alpha6"
```

Comment:
I believe it should be `aplha5` (previous version was `alpha4`)

Reply:
We are not making a release of the ink sdk. But the inkSDKVersionCode is 6. This is why I updated it here.